common/djangoapps/student/api.py

```diff
@@ -2,19 +2,17 @@
 """
 Python APIs exposed by the student app to other in-process apps.
 """
-
-
+import datetime
 import logging
 
 from django.contrib.auth import get_user_model
-from django.conf import settings
 from opaque_keys.edx.keys import CourseKey
+from pytz import UTC
 
-from common.djangoapps.student.models import CourseEnrollment
-from common.djangoapps.student.models_api import create_manual_enrollment_audit as _create_manual_enrollment_audit
-from common.djangoapps.student.models_api import get_course_access_role
-from common.djangoapps.student.models_api import get_course_enrollment as _get_course_enrollment
-from common.djangoapps.student.models_api import (
+from common.djangoapps.student.models import CourseEnrollment as _CourseEnrollment, UserProfile as _UserProfile, \
+    CourseAccessRole as _CourseAccessRole, PendingNameChange as _PendingNameChange, \
+    ManualEnrollmentAudit as _ManualEnrollmentAudit
```

Comment:
The prevailing style in edx-platform is to have one `import .... as ....` per line, preferring parentheses to backslashes. The `from common.djangoapps.student.models import (...` on the next line is a good example.

Reply:
Looks like we have docs on this, albeit old ones: https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#breaking-long-lines